libpng/pnglibconf.h

```diff
@@ -0,0 +1,219 @@
+/* pnglibconf.h - library build configuration */
+
+/* libpng version 1.6.37 */
+
+/* Copyright (c) 2018-2019 Cosmin Truta */
+/* Copyright (c) 1998-2002,2004,2006-2018 Glenn Randers-Pehrson */
+
+/* This code is released under the libpng license. */
+/* For conditions of distribution and use, see the disclaimer */
+/* and license in png.h */
+
+/* pnglibconf.h */
+/* Machine generated file: DO NOT EDIT */
```

Comment:
If this file is generated automatically, can we generate it at build time instead of adding it into the repository?

Reply:
Whether you can keep the current **pnglibconf.h** file, add a description in readme **_compile it yourself_**

Reply:
I don't think that would work. We don't support overwriting files located inside a managed component.

Okay, let's keep this file for now, we can implement build-time generation process in another PR.